### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by xuanxu**

### CODE REVIEW

File: lib/repository.rb [hunk 1]
```diff
@@ -17,4 +17,24 @@ def self.editors
...
+      nil
+    end
+
+    if invitee_id.present?
+      org = Rails.application.settings["reviews"].split("/").first
+      url = "https://api.github.com/orgs/#{org}/invitations"
+      headers = {"Authorization" => "token #{ENV['GH_TOKEN']}", "Content-Type" => "application/json", "Accept" => "application/vnd.github.v3+json"}
+      parameters = {invitee_id: invitee_id, team_ids: [config[nwo]["editor_team_id"]]}
+
+      response = Faraday.post(url, parameters.to_json, headers)
```
Comment (arfon) [inline lib/repository.rb:34]: Only issue with this in production is that Whedon isn't an org admin so this line of code currently fails.

Comment (xuanxu) [reply lib/repository.rb:34]: Right! I have to document that, the GH_TOKEN needs admin privileges

Comment (xuanxu) [reply lib/repository.rb:34]: Actualy I think Whedon is an org admin, it's already sending invitations and adding collaborators to the organization. If its token includes the admin:org scope his should not fail.

**Review by xuanxu — COMMENTED** [review]
Added fields to the new editor form: 

neweditor

File: lib/repository.rb (hunk 1 above)
Comment (arfon) [reply lib/repository.rb:34]: :+1:

**Pull request merged**